pkg/discovery/network/network.go

```diff
@@ -19,14 +19,23 @@ package network
 import (
 	"fmt"
 
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/client-go/dynamic"
 	"k8s.io/client-go/kubernetes"
+
+	submarinerclientset "github.com/submariner-io/submariner-operator/pkg/client/clientset/versioned"
+)
+
+const (
+	OperatorNamespace = "submariner-operator"
+	SubmarinerName    = "submariner"
```

Comment:
Can you create this constant in this file: subctl/operator/submarinercr/ensure.go

and then use the constant from here?